src/test/java/teammates/test/cases/common/FieldValidatorTest.java

```diff
@@ -250,51 +250,51 @@ public void testGetValidityInfoForAllowedName() {
-                        typicalFieldName,
-                        maxLength,
-                        emptyValue));
+                     "\"\" is not acceptable to TEAMMATES as a/an name field because it is empty. The value "
+                         + "of a/an name field should be no longer than 50 characters. It should not be empty.",
+                     validator.getValidityInfoForAllowedName(typicalFieldName, maxLength,
+                                                             emptyValue));
         
         ______TS("failure: untrimmed value");
         
         String untrimmedValue = " abc ";
         assertEquals("invalid: untrimmed",
-                String.format(WHITESPACE_ONLY_OR_EXTRA_WHITESPACE_ERROR_MESSAGE, typicalFieldName),
-                validator.getValidityInfoForAllowedName(
-                        typicalFieldName,
-                        maxLength,
-                        untrimmedValue));
+                     "The provided name field is not acceptable to TEAMMATES as it contains only whitespace "
+                         + "or contains extra spaces at the beginning or at the end of the text.",
+                     validator.getValidityInfoForAllowedName(typicalFieldName, maxLength,
+                                                             untrimmedValue));
     }
 
     @Test
     public void testGetInvalidityInfoForPersonName_invalid_returnSpecificErrorString() {
         String invalidPersonName = "";
         String actual = validator.getInvalidityInfoForPersonName(invalidPersonName);
         assertEquals("Invalid person name (empty) should return error message that is specific to person name",
-                     String.format(PERSON_NAME_ERROR_MESSAGE, invalidPersonName, REASON_EMPTY), actual);
+                     "\"\" is not acceptable to TEAMMATES as a/an person name because it is empty. The value "
+                         + "of a/an person name should be no longer than 100 characters. It should not be "
+                         + "empty.",
+                     actual);
     }
 
     @Test
     public void testGetInvalidityInfoForInstituteName_invalid_returnSpecificErrorString() {
         String invalidInstituteName = StringHelper.generateStringOfLength(INSTITUTE_NAME_MAX_LENGTH + 1);
         String actual = validator.getInvalidityInfoForInstituteName(invalidInstituteName);
         assertEquals("Invalid institute name (too long) should return error message that is specific to institute name",
-                     String.format(INSTITUTE_NAME_ERROR_MESSAGE, invalidInstituteName, REASON_TOO_LONG),
+                     "\"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\" is not "
+                         + "acceptable to TEAMMATES as a/an institute name because it is too long. The value "
+                         + "of a/an institute name should be no longer than 64 characters. It should not be "
+                         + "empty.",
```

Comment:
This one and the one above (both instances of `empty.`) can also go to the line before.